### PR TITLE
Added Kitchen tests and Travis

### PR DESCRIPTION
**Pull request by Martin819**

### CODE REVIEW

**Comment by cloudlab-autobuild** [comment]
Can one of the admins verify this patch?

File: bind/files/db.zone [hunk 1]
```diff
@@ -10,12 +10,14 @@ $TTL {{ zone.get('ttl', '86400') }}
                                 {{ zone.get('expire', 2419200) }}
                                 {{ zone.get('minimum', 86400) }}
                                 )
-
+{%- for type in zone.get('type', 'master') %}
```
Comment (fpytloun) [inline bind/files/db.zone]: This doesn't makes sense, you can't iterate over string.

Comment (epcim) [inline bind/files/db.zone]: I agree, @Martin819 have you mean `record.type?` on pillar?

**Comment by Martin819** [comment]
@fpytloun can you please recheck?

**Pull request merged**